lib/code_fund_web/controllers/api/ad_serve_controller.ex

```diff
@@ -5,9 +5,9 @@ defmodule CodeFundWeb.API.AdServeController do
   alias CodeFund.Schema.{Campaign, Impression, Property, Theme, Template}
 
   def embed(conn, %{"property_id" => property_id} = params) do
```

Comment:
@hopsoft why not match "template" out of params here too?

Reply:
Attempted this but get test failures similar to this when matching `template`.

```
** (Phoenix.ActionClauseError) could not find a matching CodeFundWeb.API.AdServeController.embed clause
     to process request. This typically happens when there is a
     parameter mismatch but may also happen when any of the other
     action arguments do not match. The request parameters are:

       %{"property_id" => "bae186f2-2489-43c9-8962-7f09cae083fc"}
```